### PR TITLE
Move CI from travis to GH actions

### PR DESCRIPTION
**Pull request by olix86**
Move CI from travis to GH actions

### CODE REVIEW

**Comment by alexfoias** [comment]
Info for badge : https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge

**Comment by olix86** [comment]
Badges added 👍

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+name: Python application
```
Comment (jcohenadad) [inline .github/workflows/python-app.yml]:
```suggestion
name: BIDS Validator
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # Spine Generic Public Database (Single Subject)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4299148.svg)](https://doi.org/10.5281/zenodo.4299148)
-[![Build Status](https://travis-ci.com/spine-generic/data-single-subject.svg?branch=master)](https://travis-ci.com/spine-generic/data-single-subject)
-
+![Python application](https://github.com/spine-generic/data-single-subject/workflows/Python%20application/badge.svg)
```
Comment (jcohenadad) [inline README.md]: the URL might need to be adapted with the suggested name "BIDS Validator"

Comment (olix86) [reply README.md]: done

**Review by jcohenadad — APPROVED** [review]

**Pull request merged**